src/config/parts/disables.ts

```diff
@@ -1,31 +1,34 @@
+import { join, relative } from 'pathe'
 import { GLOB_EXTS } from '../constants'
 import type { FlatConfig, NuxtESLintConfigOptions } from '../types'
 
-export default function disables(options: { dirs?: NuxtESLintConfigOptions['dirs'] }): FlatConfig[] {
-  const dirs = options.dirs ?? {}
+export default function disables(options: NuxtESLintConfigOptions): FlatConfig[] {
+  const nestedGlobPattern = ['**,', `*.${GLOB_EXTS}`] as const
 
   return [
     {
       name: 'nuxt:vue-routes-disables',
       files: [
-        '**/app.{js,ts,jsx,tsx,vue}',
-        '**/error.{js,ts,jsx,tsx,vue}',
-        // These pages are not used directly by users so they can have one-word names.
-        ...dirs.pages?.map(dir => `${dir}/**/*.${GLOB_EXTS}`) ?? [],
-        ...dirs.layouts?.map(dir => `${dir}/**/*.${GLOB_EXTS}`) ?? [],
+        relative(options.srcDir || '/', `app.${GLOB_EXTS}`),
+        relative(options.srcDir || '/', `error.${GLOB_EXTS}`),
```

Comment:
I haven't tested it, but would ESLint take '/app.vue' as relative to the user root, or system root? Would './app.vue' make more sense?